vertx-mail-client/src/test/java/io/vertx/ext/mail/MailLocalTest.java

```diff
@@ -49,15 +50,24 @@ public void mailTestTLSTrustAll(TestContext testContext) {
   public void mailTestTLSNoTrust(TestContext testContext) {
     this.testContext=testContext;
     MailClient mailClient = MailClient.createNonShared(vertx, configLogin().setStarttls(StartTLSOptions.REQUIRED));
-    testException(mailClient, exampleMessage());
+    testException(mailClient, DecoderException.class);
   }
 
   @Test
   public void mailTestTLSCorrectCert(TestContext testContext) {
-    this.testContext=testContext;
-    MailClient mailClient = MailClient.createNonShared(vertx,
-      configLogin().setStarttls(StartTLSOptions.REQUIRED).setKeyStore("src/test/resources/certs/keystore.jks")
-        .setKeyStorePassword("password"));
+    this.testContext = testContext;
+    final MailConfig config = configLogin().setStarttls(StartTLSOptions.REQUIRED)
+        .setKeyStore("src/test/resources/certs/client.jks").setKeyStorePassword("password");
+    MailClient mailClient = MailClient.createNonShared(vertx, config);
+    testSuccess(mailClient, exampleMessage(), assertExampleMessage());
+  }
+
+  @Test
+  public void mailTestTLSCase(TestContext testContext) {
+    this.testContext = testContext;
+    final MailConfig config = configLogin().setHostname("LOCALHOST").setStarttls(StartTLSOptions.REQUIRED)
+        .setKeyStore("src/test/resources/certs/client.jks").setKeyStorePassword("password");
```

Comment:
minor : you should be able to use _certs/client.jks_ directly as keystores are also loaded from classpath